bots/configure-bot-environment/README.md

```diff
@@ -1,57 +1,44 @@
-`LISTEN_ON_ALL_PUBLIC` | Stream callbacks receive messages from all public channels (true/false). Defaults to `false` (only rooms the bot has joined).
-`ROCKETCHAT_ROOM`      | Stream callbacks receive messages from these channel name/s. Defaults to `GENERAL`. Accepts comma separated list.
-`RESPOND_TO_LIVECHAT`  | Stream callbacks receive messages from Livechat (true/false). Defaults to `false`.
-`RESPOND_TO_DM`        | Stream callbacks receive DMs with bot (true/false). Defaults to `false`.
-`RESPOND_TO_EDITED`    | Stream callbacks receive edited messages (true/false). Defaults to `false`.
+`LISTEN_ON_ALL_PUBLIC` | Stream callbacks receive messages from all public channels (true/false). Default: `false` (only rooms the bot has joined).
+`ROCKETCHAT_ROOM`      | Stream callbacks receive messages from the listed channel name/s. Default: `GENERAL`. Accepts a comma separated list. Allows the bot to listen and respond to messages _from all newly created private groups_ where the bot's user has been added as a member. Should be empty if `LISTEN_ON_ALL_PUBLIC=true`
+`RESPOND_TO_LIVECHAT`  | Stream callbacks receive messages from Livechat (true/false). Default: `false`.
+`RESPOND_TO_DM`        | Stream callbacks receive DMs with bot (true/false). Default: `false`.
+`RESPOND_TO_EDITED`    | Stream callbacks receive edited messages (true/false). Default: `false`.
 **SDK Development**    |
 `ROOM_CACHE_SIZE`      | Size of cache (LRU) for room (ID or name) lookups.
 `ROOM_CACHE_MAX_AGE`   | Max age of cache for room lookups.
 `DM_ROOM_CACHE_SIZE`   | Size of cache for Direct Message room lookups.
 `DM_ROOM_CACHE_MAX_AGE`| Max age of cache for DM lookups.
-`INTEGRATION_ID`       | ID applied to message object to integration source. Defaults to `js.SDK`
-`ADMIN_USERNAME`       | Admin user password for API calls used in SDK tests.
+`INTEGRATION_ID`       | ID applied to message object to integration source. Default: `js.SDK`
+`ADMIN_USERNAME`       | Admin user name for API calls used in SDK tests.
 `ADMIN_PASS`           | Admin user password for API calls used in SDK tests.
 **Hubot Specific**     |
 `HUBOT_NAME`           | Name of the bot. Hubot listeners can respond to this.
 `HUBOT_ALIAS`          | Another name to respond to. If unset, the adapter sets the `ROCKETCHAT_USER` as an alias, to ensure bots respond when addressed using their username.
 `EXTERNAL_SCRIPTS`     | Hubot scripts to require as NPM modules. Used only in some Docker instances.
-`HUBOT_LOG_LEVEL`      | `debug`, `info`, `warning` or `error`. Default `info`.
+`HUBOT_LOG_LEVEL`      | `debug`, `info`, `warning` or `error`. Default: `info`.
 
-## Common configuration
+## Common set of variables
 
-It is common to set up a bot to listen and respond to direct messages and all
-new public channels and private groups. Use the following options:
+Most of the bots you create will listen and respond to direct messages and all new public channels and
```

Comment:
> Most of the bots you create will respond

This seems odd because we say the bot can respond to DMs public and private channels, but then split the settings up leaving Respond to DM duplicated here and moving the other two setting to the list above?

 I agree it isn't bad to get all the settings in one list, but if you say 'bot can do x'  you need to illustrate the settings that you need to do x ?

Could you split your variable list with something like this as you have with say Hubot?:

**Responding  to channels and DMS**

`RESPOND_TO_LIVECHAT` 
`RESPOND_TO_DM`  
`RESPOND_TO_EDITED`

Reply:
Agreed. Fixed.